dns/OWNERS

```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - durandom
+  - goern
+  - schwesig
+
+reviewers:
+  - tumido
```

Comment:
Can you move me to `approvers`, please?

Reply:
ja, was not sure about the amount of involvement you want ;)